.golangci.yml

```diff
@@ -0,0 +1,53 @@
+# Reference of settings:
+# https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml
+
```

Comment:
Don't we want to skip dir rest/models from linting?

Reply:
Generated files are not linted by default.